.nycrc

```diff
@@ -0,0 +1,3 @@
+{
+    "temp-directory": "coverage"
+}
```

Comment:
we don't seem to be using this in our other projects that use code coverage. what's the reason for having this file?

Reply:
By default nyc reads coverage data from it's own temp directory (`./.nyc_output` [istanbul/nyc lib/config-util.js:215](https://github.com/istanbuljs/nyc/blob/2030e0c6721f574500a745277334b8bf20d58ccb/lib/config-util.js#L215)), while ember-cli-code-coverage writes to `./coverage`.